docs/Quick-Start-Guide/Architecture.md

```diff
@@ -257,43 +271,43 @@ to go through is **fuse translator** which falls under the category of
 
 1.  **Cluster Translators**:
 
-    *DHT(Distributed Hash Table)
+    *DHT(Distributed Hash Table)
```

Comment:
formatting is a bit of a miss for DHT and AFR

Reply:
I will address these things in separate PR. There was a special char was used instead of space I removed that.